src/IROperator.h

```diff
@@ -1907,6 +1906,18 @@ Expr saturating_cast(Expr e) {
  * maximum values of the result type. */
 Expr saturating_cast(Type t, Expr e);
 
+/** Makes a best effort attempt to preserve IEEE floating-point semantics
```

Comment:
Is this comment still accurate? no_float_simplify isn't a flag or intrinsic anymore.